pymisp/abstract.py

```diff
@@ -37,6 +39,53 @@ def tzname(self, dt):
         def dst(self, dt):
             return timedelta(0)
 
+    class MISPFileCache(object):
+        # cache up to 150 JSON structures in class attribute
+        __file_cache = cachetools.LFUCache(150)
+
+        @classmethod
+        def _load_json(cls, path):
```

Comment:
This could be aligned with the Python 3 approach by using the cachetools decorators:
https://cachetools.readthedocs.io/en/stable/#module-cachetools.func

Reply:
We could do that, I need to check how that works, because I don't have python2 anymore.